sqlitecluster/SQLiteUtils.h

```diff
@@ -0,0 +1,14 @@
+#pragma once
+
+class SQLiteUtils {
+  public:
+      /**
+       * Gets the next ID available for random IDs on a given table and column
+       *
+       * @param db
+       * @param tableName
+       * @param column
+       * @return the new ID
```

Comment:
Also we stopped doing the javadoc style param/return stuff for C++ where the code itself does the exact same thing.